lxd/instance/drivers/driver_qemu.go

```diff
@@ -5,8 +5,10 @@ import (
 	"bytes"
 	"compress/gzip"
 	"context"
+	"crypto/sha1"
```

Comment:
This is the only place where `crypto/sha1` is used. Would it be a candidate for using SHA-256 that is trimmed down instead?